PENDING_PENDING_RELEASE_NOTES.md

```diff
@@ -10,5 +10,14 @@ _Date TBD_
   * Added new Quartz job to gather and store Summary Statistics
   * Added new user schedulable job that sends the Summary Statistics Email
   * Added new endpoint for retrieving all jobs that user has permission to schedule
+* Download XML file changes
+  * In the <ucdProces> node
+    * AllVersions will now be structured: `<allVersions><version/><version/> ... </allVersions>`
+	* SuccessVersions will now be structured: `<successVersions><version/><version/> ... </successVersions>`
+  * In the <listing> node
+    * CertificationStatusEvents will now be structured: `<certificationEvents><certificationEvent><eventDate/><id/><reason/></status></certificationEvent> ... </certificationEvents>`
+	* TestingLabs will now be structured: `<testingLabs><testingLabs/><testingLab/> ... </testingLabs>`
+  * In the <ucdProcess> node
+    * Criteria will no be structured: `<criteriaList><criteria/><criteria/> ... </criteriaList>`
```

Comment:
no -> now

Also, above, are AllVersions and SuccessVersions on a different level of indentation? If so, why?

Similar question with CertificationStatusEvents and TestLabs

Also, if the xml file is changing structure, we'll need that XSD changed file indicator in the UI project